examples/experiment_eval.py

```diff
@@ -108,6 +108,12 @@ def _iterate_env_agent_combo_not_in_db(
     env_ids: list[str] = [],
     tag: str | None = None,
 ) -> Generator[EnvAgentCombo[Observation, AgentAction], None, None]:
+    evaluation_dimensions = (
+        EvaluationDimensionGenerator.generate_dimension_model_from_name(
```

Comment:
Instead of naming `EvaluationDimensionGenerator`, can we do `EvaluationDimensionBuilder`? 

`generate_dimension_model_from_name` -> `select_existing_dimension_model_by_name`?